docs/en/latest/admin-api.md

```diff
@@ -427,37 +422,37 @@ After successful execution, upstream nodes will not retain the original data, an
+| DELETE | /apisix/admin/consumers/{username} | NULL         | Removes the Consumer with the specified username. |
 
 ### Request Body Parameters
 
-| Parameter   | Required | Type        | Description                                                         | Example                                          |
-| ----------- | -------- | ----------- | ------------------------------------------------------------------- | ------------------------------------------------ |
-| username    | True     | Name        | Consumer name                                                       |                                                  |
-| plugins     | False    | Plugin      | See [Plugin](architecture-design/plugin.md) for more                |                                                  |
-| desc        | False    | Auxiliary   | Identifies route names, usage scenarios, and more.                  | customer xxxx                                    |
-| labels      | False    | Match Rules | Key/value pairs to specify attributes                               | {"version":"v2","build":"16","env":"production"} |
-| create_time | False    | Auxiliary   | epoch timestamp in second, will be created automatically if missing | 1602883670                                       |
-| update_time | False    | Auxiliary   | epoch timestamp in second, will be created automatically if missing | 1602883670                                       |
+| Parameter   | Required | Type        | Description                                                                                                        | Example                                          |
+| ----------- | -------- | ----------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------ |
+| username    | True     | Name        | Name of the Consumer.                                                                                              |                                                  |
+| plugins     | False    | Plugin      | Plugins that are executed during the request/response cycle. See [Plugin](architecture-design/plugin.md) for more. |                                                  |
+| desc        | False    | Auxiliary   | Description of usage scenarios.                                                                                    | customer xxxx                                    |
+| labels      | False    | Match Rules | Attributes of the Consumer specified as key-value pairs.                                                           | {"version":"v2","build":"16","env":"production"} |
+| create_time | False    | Auxiliary   | Epoch timestamp in second of the created time. If missing, this field will be populated automatically.             | 1602883670                                       |
```

Comment:
> Epoch timestamp in second of the created time.

I am bit confused on the framing of this sentence. How about this -


```suggestion
| create_time | False    | Auxiliary   | Epoch timestamp (in seconds) of the created time. If missing, this field will be populated automatically.             | 1602883670                                       |
```

Reply:
Yes. I missed this.